server/api/impl/configurator/configurator.go

```diff
@@ -1,20 +1,18 @@
 package configurator
 
 import (
+	"encoding/json"
 	"net/url"
 	"strings"
 	"sync"
 
-	"github.com/mattermost/mattermost-plugin-apps/server/api"
-	"github.com/mattermost/mattermost-plugin-apps/server/api/impl/aws"
-
 	"github.com/pkg/errors"
 
 	pluginapi "github.com/mattermost/mattermost-plugin-api"
 	"github.com/mattermost/mattermost-server/v5/model"
-)
 
-var _ api.Configurator = (*config)(nil)
```

Comment:
Don't we want to check that any longer?

Reply:
It is effectively checked in https://github.com/mattermost/mattermost-plugin-apps/pull/85/files#diff-42a3dc1fa0c237add93a7841342bdb1ab36a39db77a8c7adfc54a87ad654fc3cR28